src/app/settings/page.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+
+import { useRouter } from 'next/navigation';
+import supabase from '../../api/supabase/createClient';
+
+export default function Settings() {
+    const { push } = useRouter();
+    const signOut = async () => {
+        const { error } = await supabase.auth.signOut()
+        if (error) {
+            throw new Error(`An error occurred trying to sign out: ${error}`);
+          }
+        push("/login");
+    } 
+    const resetPassword = () => {
+        push("/reset-password");
+    }
+
+    return (
+        <>
+            <button type="button" onClick={signOut}>Sign out</button>
+            <button type="button" onClick={resetPassword}>Reset Password</button>
```

Comment:
We can just do `onClick={() => push('/reset-password')}` as the `resetPassword` handler is used only for routing and doesn't actually help reset the password.

Reply:
@pragyakallanagoudar in the future, are we going to use a more advanced reset-password involving email redirect link? if so, do we want to keep this function?

Reply:
To allow for this possibility, we'll leave the handler function for right now!